libs/langchain-mongodb/src/vectorstores.ts

```diff
@@ -256,6 +256,22 @@ export class MongoDBAtlasVectorSearch extends VectorStore {
     });
   }
 
+  /**
+   * Delete documents from the collection
+   * @param ids - An array of document IDs to be deleted from the collection.
+   *
+   * @returns - A promise that resolves when all documents deleted
+   */
+  async delete(params: { ids: any[] }): Promise<void> { // eslint-disable-line @typescript-eslint/no-explicit-any
+    const splice = <T>(l: T[], c: number): T[][] =>
```

Comment:
Can we use the built-in `chunkArray` method instead?

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-openai/src/embeddings.ts#L211